cli/cli_app.go

```diff
@@ -233,7 +233,7 @@ func downloadModules(terragruntOptions *options.TerragruntOptions) error {
 			return err
 		}
 		if shouldDownload {
-			return shell.RunTerraformCommand(terragruntOptions, "get", "-update")
+			return shell.RunTerraformCommandAndIgnoreOutput(terragruntOptions, "get", "-update")
```

Comment:
I completely agree about not polluting output, but the way to do that is not to throw the output away, but to use different logging levels or, as you said, to format the output differently.